kw.sh

```diff
@@ -5,9 +5,53 @@ EASY_KERNEL_WORKFLOW=${EASY_KERNEL_WORKFLOW:-"kw"}
 src_script_path=${src_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/src"}
 external_script_path=${external_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/external"}
 
+# Notification definitions
+declare -r ALERT_NONE=0
+declare -r ALERT_VISUAL=1
+declare -r ALERT_SOUND=2
+declare -r ALERT_BOTH=3
+
 # Export external variables required by kworkflow
 export EASY_KERNEL_WORKFLOW
 
+# Alerts command completion to the user. First argument is the alert code,
+# and second is the command string to be printed in visual notification.
+function alert_completion()
+{
+  local ALERT=$1
+  local COMMAND=$2
+  if [[ $(($ALERT & $ALERT_SOUND)) != 0 ]]; then
+    echo -e "\a\a"
+  fi
+  if [[ $(($ALERT & $ALERT_VISUAL)) != 0 ]]; then
+    notify-send -i checkbox -t 10000 "kw" "Command: \"$2\" completed!"
+  fi
+}
+
+# Check the options for "--alert=[v|s|vs|sv|n]" and returns the respectively
+# alert code.
+function check_alert_option()
+{
+  if [[ $# -ge 1 && "$1" =~ ^--alert= ]]; then
+    local opt="$(echo $1 | sed s/--alert=// )"
+    case "$opt" in
+      v)
+        return $ALERT_VISUAL
+        ;;
+      s)
+        return $ALERT_SOUND
+        ;;
+      vs|sv)
+        return $ALERT_BOTH
+        ;;
+      n)
+        return $ALERT_NONE
+        ;;
+     esac
+  fi
+  return $ALERT_NONE
```

Comment:
Take care with the use of return in shell script, the use of it is restricted. Basically, return just inform values from 0-255 which represents the errno code. In shell just use return for return status.
If you want to get some value, I recommend you to do something like this:
```sh
function lala()
{
 ...
tmp="xpto"
echo $tmp
 ...
}
...
value=$(lala)
echo $value
...
```

This approach provide an extra advantage for you: you can save one line and one variable in the kw() function. You can do something like:

```sh
local alert=$(lala $@)
```

However, I'm asking to myself: can we simplify this code?